### ISSUE TITLE
Flight Replay example uses incorrect API

### ISSUE THREAD

**Issue by hamishwillee** [diydrones/dronekit-python#66]
The [Flight Replay example](http://python.dronekit.io/example_3.html#setting-the-new-waypoints) sets waypoints using mavutil.mavlink.MAVLink_mission_item_message directly. As I understand the API the correct type for a command in this API is "Command" (which is not even imported)

```
    cmd = mavutil.mavlink.MAVLink_mission_item_message(0,
                                                     0,
                                                     0,
                                                     mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT,
                                                     mavutil.mavlink.MAV_CMD_NAV_WAYPOINT,
                                                     0, 0, 0, 0, 0, 0,
                                                     lat, lon, altitude)
    cmds.add(cmd)
```

To fix this I believe the change is trivial, because the Command is just a thin wrapper around MAVLink_mission_item_message

```

from droneapi.lib import VehicleMode, Command
...
    cmd = Command(0,
                                                     0,
                                                     0,
                                                     mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT,
                                                     mavutil.mavlink.MAV_CMD_NAV_WAYPOINT,
                                                     0, 0, 0, 0, 0, 0,
                                                     lat, lon, altitude)
    cmds.add(cmd)
````

**Comment by mrpollo** [comment]
@hamishwillee did you try this in SITL?

**Comment by hamishwillee** [comment]
I ran both the original code and with my changes above. In both cases the output "looked reasonable" and was the same. 

BUT, I did NOT go through the code fully yet and confirm that it is all working as expected. So "looking reasonable" is not a canonical "this code works perfectly".

**Comment by hamishwillee** [comment]
Tried this in SITL and the fixed file works. I have made this change. Closing as fix is in #81
